integration

```diff
@@ -67,6 +67,7 @@
   "ankohanse/hass-dab-pumps",
   "Anonym-tsk/homeassistant-climate-xiaomi-remote",
   "anrolosia/shopping-list-with-grocy",
+  "antibill51/rika-firenet-custom-component",
```

Comment:
The repo description is in French; if this is a French market only, this needs to be noted in hacs.json.
If not then the description should be in English.